list/List.js

```diff
@@ -182,6 +182,8 @@ define([
 
 		template: template,
 
+		tabindex: 0,
```

Comment:
I have renamed the `tabindex` property with `tabIndex` but this generate errors while parsing the template. I guess because tabIndex is reserved keyword.